projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html

```diff
@@ -30,23 +30,21 @@
     {{ callNumber }}
   </div>
   <div class="col-sm-4 text-right">
-    <ng-container *ngIf="isHoldingMatchUserLibraryPID">
```

Comment:
diff is not always user friendly... in this case I only removed `<ng-container *ngIf="isHoldingMatchUserLibraryPID">`. The button is now based only on permissions.